install/prereqs.rst

```diff
@@ -0,0 +1,93 @@
+#############
+Prerequisites
+#############
+
+This page lists software needed to install and use the LSST Science Pipelines.
+
+.. _prereq-platforms:
+
+Platform compatibility
+======================
+
+The LSST Data Management reference platform is CentOS 7.
+This is the platform we officially develop, test, and operate with.
+
+Besides the reference platform, our developers and automatic tests regularly compile and run the Pipelines under CentOS 8, Debian and Ubuntu Linux, and macOS.
```

Comment:
I'm not sure Debian and Ubuntu should be considered "regularly".  Even ls.st/faq only shows Ubuntu, not "basic" Debian.

Reply:
Changed to:

> Besides the reference platform, our developers and automatic tests regularly compile and run the Pipelines under CentOS 8, a variety of other Linux distributions, and various versions of macOS.